service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/TemporaryHearingJudgeContentProvider.java

```diff
@@ -20,9 +21,23 @@
 
     public TemporaryHearingJudgeTemplate buildNotificationParameters(CaseData caseData, HearingBooking hearingBooking) {
         JudgeAndLegalAdvisor judgeAndLegalAdvisor = hearingBooking.getJudgeAndLegalAdvisor();
+
+        String judgeTitle = judgeAndLegalAdvisor.getJudgeOrMagistrateTitle();
+        String judgeName = judgeAndLegalAdvisor.getJudgeName();
+
+        if (judgeAndLegalAdvisor.getJudgeTitle().equals(MAGISTRATES)) {
```

Comment:
Will we have this issue else where when we are notifying a judge? I don't remember seeing any special logic for judge names anywhere else.

Reply:
I'll check that one

Reply:
<img width="1231" alt="image" src="https://user-images.githubusercontent.com/15530123/101771019-379caf00-3ae1-11eb-85b5-03b23fdcb858.png">
cant have allocated magistrate judge with no name